src/libpgmoneta/wal.c

```diff
@@ -508,6 +522,16 @@ pgmoneta_wal(int srv, char** argv)
    exit(1);
 }
 
+char*
+wal_xlogptr_to_xlogpos(size_t xlogptr)
```

Comment:
Rename to `update_wal_lsn` and update the function signature

Reply:
Updated it. Instead of returning a value, I assigned a new value to `config->servers[srv].current_wal_lsn` inside the function. It can avoid memory leaks as well.